src/components/fileUpload.js

```diff
@@ -180,6 +192,171 @@
       </div>
     );
 
+    const Hr = () => <hr className={classes.hr} />;
+
+    const DeleteButton = ({ file }) => (
+      <div className={classes.deleteButtonWrapper}>
+        <IconButton
+          size="small"
+          className={classes.remove}
+          onClick={file ? () => removeFileFromList(file.url) : {}}
```

Comment:
Are you sure `onClick` can receive an object as value? I think the if statement should be moved inside the function like this:

```javascrip
onClick={() => {
  if(file) {
    removeFileFromList(file.url);
  }
}}
```